### ISSUE TITLE
Pretty formatter outputting blank lines after hook methods

### ISSUE THREAD

**Issue by hackel** [Behat/Behat#1200]
Today I was working in Behat (3.5.0) and everything was fine, then suddenly the output of the pretty formatter changed to insert all of these blank spaces with ascii lines around hook methods.  e.g.
```
┌─ @BeforeSuite # FeatureContext::someBeforeSuiteHook()
│
│  
│  
│

Feature: Some feature
    ...

  ┌─ @BeforeScenario # FeatureContext::someBeforeScenarioHook()
  │
  │  
  │  
  │
  Scenario: First scenario...
```
I've never seen this empty hook output before, and I have no idea what I changed.  I've gone through all of my code changes, and nothing seems to be causing this.  Just trying to figure out how to switch it back to normal.  Any idea what might be going on?

**Comment by ciaranmcnulty** [comment]
That’s how we present output from hooks. Is it possible the hook is outputting some new lines / blank strings?

**Comment by hackel** [comment]
That's what I suspected, and I've searched my code for anything that might be inadvertently outputting something but haven't found anything.  I'll keep looking, it's just odd since this happened so suddenly and I can see all of the changes made in my working tree.  Haven't updated any dependencies either.  Could just be an obscure typo in one of my dependencies that got saved by mistake.  I'll try reinstalling them as well.
